### PR TITLE
New .Net Core client

### PR DESCRIPTION
**Pull request by rigazilla**
Blog post to introduce the Hotrod .Net Core client

### CODE REVIEW

**Comment by karesti** [comment]
@oraNod some time for a quick review ?

File: _posts/2021-06-01-dotnet-core-client.adoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+author: rigazilla
+tags: [ "hotrod", ".Net Core", "client" "C#" ]
+---
+Dear Infinispanners,
+
+we'd like to share with you a new project the Infinispan Team is working on: https://github.com/infinispan/Infinispan.Hotrod.Core[Infinispan.Hotrod.Core].
+
+As you may know the current .Net client is based on the {cpp} core implementation, this architectural solution has pros and cons:
+features can be easily ported between {cpp} and C# (this has really guarateed features parity beetween the two clients over the time) but as a drawback:
+it ties the client to a specific architecture, complicating portability and distribution.
```
Comment (oraNod) [inline _posts/2021-06-01-dotnet-core-client.adoc:14]:
```suggestion
Our current .NET client is based on the {cpp} core implementation, which is an solution that has its pros and cons.
It makes it easier to guarantee parity between the {cpp} and C# clients over the time, but has the drawback of tying clients to specific a architecture. 
In turn that complicates portability and distribution, making the release lifecycle for those clients more onerous and sluggish.
```

File: _posts/2021-06-01-dotnet-core-client.adoc [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+The https://github.com/infinispan/Infinispan.Hotrod.Core[Infinispan.Hotrod.Core] is a 100% C# Hotrod client designed with the aim to be portable across
```
Comment (oraNod) [inline _posts/2021-06-01-dotnet-core-client.adoc:16]:
```suggestion
The https://github.com/infinispan/Infinispan.Hotrod.Core[Infinispan.Hotrod.Core] is a 100% C# Hot Rod client designed with the aim of being portable across
```

File: _posts/2021-06-01-dotnet-core-client.adoc [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+systems via the .Net Core platform and easy to deploy and consume via the Nuget platform.
```
Comment (oraNod) [inline _posts/2021-06-01-dotnet-core-client.adoc:17]:
```suggestion
systems via the .Net Core platform and easier to deploy and consume via the Nuget platform.
```

File: _posts/2021-06-01-dotnet-core-client.adoc [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+If you live in the .Net Core ecosystem you'll probably want to look at it. Entry points are:
+
+* https://github.com/infinispan/Infinispan.Hotrod.Core[Infinispan.Hotrod.Core], current status and news;
+* https://github.com/infinispan/Infinispan.Hotrod.Core/tree/main/Infinispan.Hotrod.Core.XUnitTest[XUnit test], this will contain the same testsuite of the
+current https://github.com/infinispan/dotnet-client[.Net client] project, so to ease maturity comparison;
```
Comment (oraNod) [inline _posts/2021-06-01-dotnet-core-client.adoc:23]:
```suggestion
current https://github.com/infinispan/dotnet-client[.NET client] project with the aim of making it easier to compare with the .NET core client as it matures.
```

File: _posts/2021-06-01-dotnet-core-client.adoc [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+* https://github.com/infinispan/Infinispan.Hotrod.Core/tree/main/Infinispan.Hotrod.Application[Sample Application], an example of project which uses the client package;
```
Comment (oraNod) [inline _posts/2021-06-01-dotnet-core-client.adoc:24]:
```suggestion
* https://github.com/infinispan/Infinispan.Hotrod.Core/tree/main/Infinispan.Hotrod.Application[Sample Application], an example of project which uses the client package.
```

File: _posts/2021-06-01-dotnet-core-client.adoc [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+* https://www.nuget.org/packages/Infinispan.Hotrod.Core/[Nuget], where the package is published.
+
+Hope this will makes C# developers happy!
+
+Please let us know your thoughts, good place for them is the https://github.com/infinispan/Infinispan.Hotrod.Core/issues[issues] page.
```
Comment (oraNod) [inline _posts/2021-06-01-dotnet-core-client.adoc:29]:
```suggestion
Please let us know your thoughts, a good place for them is the https://github.com/infinispan/Infinispan.Hotrod.Core/issues[issues] page.
```

File: _posts/2021-06-01-dotnet-core-client.adoc [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+The https://github.com/infinispan/Infinispan.Hotrod.Core[Infinispan.Hotrod.Core] is a 100% C# Hotrod client designed with the aim to be portable across
+systems via the .Net Core platform and easy to deploy and consume via the Nuget platform.
+
+If you live in the .Net Core ecosystem you'll probably want to look at it. Entry points are:
+
+* https://github.com/infinispan/Infinispan.Hotrod.Core[Infinispan.Hotrod.Core], current status and news;
+* https://github.com/infinispan/Infinispan.Hotrod.Core/tree/main/Infinispan.Hotrod.Core.XUnitTest[XUnit test], this will contain the same testsuite of the
+current https://github.com/infinispan/dotnet-client[.Net client] project, so to ease maturity comparison;
+* https://github.com/infinispan/Infinispan.Hotrod.Core/tree/main/Infinispan.Hotrod.Application[Sample Application], an example of project which uses the client package;
+* https://www.nuget.org/packages/Infinispan.Hotrod.Core/[Nuget], where the package is published.
```
Comment (oraNod) [inline _posts/2021-06-01-dotnet-core-client.adoc:25]:
```suggestion
* https://www.nuget.org/packages/Infinispan.Hotrod.Core/[Nuget] is where the package is published.
```

File: _posts/2021-06-01-dotnet-core-client.adoc [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+we'd like to share with you a new project the Infinispan Team is working on: https://github.com/infinispan/Infinispan.Hotrod.Core[Infinispan.Hotrod.Core].
+
+As you may know the current .Net client is based on the {cpp} core implementation, this architectural solution has pros and cons:
+features can be easily ported between {cpp} and C# (this has really guarateed features parity beetween the two clients over the time) but as a drawback:
+it ties the client to a specific architecture, complicating portability and distribution.
+
+The https://github.com/infinispan/Infinispan.Hotrod.Core[Infinispan.Hotrod.Core] is a 100% C# Hotrod client designed with the aim to be portable across
+systems via the .Net Core platform and easy to deploy and consume via the Nuget platform.
+
+If you live in the .Net Core ecosystem you'll probably want to look at it. Entry points are:
```
Comment (oraNod) [inline _posts/2021-06-01-dotnet-core-client.adoc:19]:
```suggestion
If you are involved with the .NET Core ecosystem we hope you will find this project of interest. Entry points for the project are:
```

**Review by oraNod — COMMENTED** [review]
@rigazilla this got a little lost in my inbox. sorry for delay. I made some suggestions, please feel free to reword or ignore. this sounds like a pretty cool project. I'll have to check out Nuget.

**Comment by rigazilla** [comment]
@oraNod, I've commited you suggestions, thanks!

**Comment by oraNod** [comment]
nice one @rigazilla ! glad those suggestions were useful too.

**Pull request merged**